api/cluster.go

```diff
@@ -52,7 +53,7 @@ func (s *Server) indexFind(ctx *middleware.Context, req models.IndexFind) {
 	for _, pattern := range req.Patterns {
 		nodes, err := s.MetricIndex.Find(req.OrgId, pattern, req.From)
 		if err != nil {
-			response.Write(ctx, response.NewError(http.StatusBadRequest, err))
+			response.Write(ctx, response.NewError(http.StatusBadRequest, err.Error()))
```

Comment:
my bad. I got it confused with fmt.printf style functions taking error values as strings for `%s`